tools/serverpod_cli/lib/src/util/command_line_tools.dart

```diff
@@ -167,3 +144,24 @@ class _CommandFormatter {
     return 'CMD: $command ${args.join(' ')}';
   }
 }
+
+Future<int> _runProcessWithLoggerAttached({
```

Comment:
Naming isn't great as the logger isn't attached (which sounds like it is passed as an argument). `_runProcessWithDefaultLogger` would be better.